src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -118,6 +118,21 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     return !this.pageValidationService.isPageValid(this.currentPage, this.editForm);
   }
 
+  toPreviousPage() {
```

Comment:
this seems like a big change in the behaviour of the previous method. I wonder whether the current behaviour was like that by design. 
What we can do is we can approve these changes but ask QAs to do full regression testing on the behaviour of navigating to previous pages.